var/spack/repos/builtin/packages/py-htseq/package.py

```diff
@@ -10,11 +10,20 @@ class PyHtseq(PythonPackage):
     """HTSeq is a Python package that provides infrastructure to process
     data from high-throughput sequencing assays."""
 
-    homepage = "https://htseq.readthedocs.io/en/release_0.9.1/overview.html"
-    url = "https://github.com/simon-anders/htseq/archive/release_0.9.1.tar.gz"
+    homepage = "https://htseq.readthedocs.io/en/master/index.html"
+    url = "https://github.com/htseq/htseq/archive/release_0.12.3.tar.gz"
 
-    version("0.11.2", sha256="dfc707effa699d5ba9034e1bb9f13c0fb4e9bc60d31ede2444aa49c7e2fc71aa")
-    version("0.9.1", sha256="28b41d68aa233fce0d57699e649b69bb11957f8f1b9b7b82dfe3415849719534")
+    version("0.12.3", sha256="d4710758dd39be2b37a870945b813c6c9e50a1821863c564bf4504636a7eb7a3")
+    version(
+        "0.11.2",
+        url="https://github.com/simon-anders/htseq/archive/release_0.11.2.tar.gz",
+        sha256="dfc707effa699d5ba9034e1bb9f13c0fb4e9bc60d31ede2444aa49c7e2fc71aa",
+    )
+    version(
+        "0.9.1",
+        url="https://github.com/simon-anders/htseq/archive/release_0.9.1.tar.gz",
+        sha256="28b41d68aa233fce0d57699e649b69bb11957f8f1b9b7b82dfe3415849719534",
+    )
 
     depends_on("py-setuptools", type="build")
     depends_on("py-numpy", type=("build", "run"))
```

Comment:
Cython and swig are not required at run-time, and matplotlib is an optional dependency

Reply:
Updating to a newer version from where I'd checksummed to before, there are other variant dependencies as well, so have re-done this whole chunk in a new commit.